app/routers/kis_domestic_trading.py

```
@@ -0,0 +1,191 @@
+async def analyze_my_domestic_stocks():
+    """보유 국내 주식 AI 분석 실행 (Celery)"""
+    try:
+        from app.core.celery_app import celery_app
+        async_result = celery_app.send_task("kis.run_analysis_for_my_domestic_stocks")
+
+        return {
+            "success": True,
+            "message": "국내 주식 분석이 시작되었습니다.",
+            "task_id": async_result.id
+        }
+    except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)